src/pymovements/datasets/daemons.py

```diff
@@ -103,6 +106,8 @@ class DAEMONS(DatasetDefinition):
 
     name: str = 'DAEMONS'
 
+    long_name: str = 'DAta set for Eye Movement On Natural Scenes'
```

Comment:
judging from the publication I think it should be named: `Potsdam data set of eye movement on natural scenes`

Reply:
done